ts/create-live-compositor/templates/node-express-redux/src/store.ts

```diff
@@ -0,0 +1,40 @@
+import type { Action, ThunkAction } from "@reduxjs/toolkit"
+import { combineSlices, configureStore, createSlice } from "@reduxjs/toolkit"
+
+export const showInstructionsSlice = createSlice({
+  name: "instructions",
+  initialState: true,
+  reducers: create => ({
+    toggle: create.reducer(state => !state)
+  }),
+  selectors: {
+    shouldShow: state => state,
+  }
+})
+
+// `combineSlices` automatically combines the reducers using
```

Comment:
all of the comments here are from redux template, I decided to left them, but I don't have a strong opinon on that